lib/rouge/cli.rb

```diff
@@ -200,9 +200,14 @@ def self.doc
         yield %[                            delimiters. implies --escape]
       end
 
+      # TODO: find a better way to do this?
+      def self.supports_truecolor?
+        ENV['TERM'] == 'xterm' || %w(24bit truecolor).include?(ENV['COLORTERM'])
```

Comment:
It doesn't look like there's an especially good way to detect this in Ruby. The best I could find was [this test](https://github.com/janlelis/paint/blob/879c0a40d05563e57e7c091d2d5a96c3f5cc82a4/lib/paint.rb#L172-L193) in the Paint gem. What do you think?